libs/sdk-ui-kit/src/Dialog/StylingEditorDialog/StylingEditorDialog.tsx

```diff
@@ -0,0 +1,160 @@
+// (C) 2022 GoodData Corporation
+import React, { useMemo, useState } from "react";
+import { IDialogBaseProps } from "../typings";
+import { Dialog } from "../Dialog";
+import { Typography } from "../../Typography";
+import cx from "classnames";
+import { StylingExample } from "./StylingExample";
+import { IThemeMetadataObject } from "@gooddata/sdk-model";
+import { BubbleHeaderSeparator } from "./BubbleHeaderSeparator";
+import { StylingEditorDialogFooter, IStylingEditorDialogFooterProps } from "./StylingEditorDialogFooter";
+import { IntlWrapper } from "@gooddata/sdk-ui";
+import { useIntl } from "react-intl";
+import { Message } from "../../Messages";
+
+/**
+ * @internal
+ */
+export interface IStylingEditorDialogProps extends IDialogBaseProps, IStylingEditorDialogFooterProps {
+    title: string;
+    tooltip?: string;
+    stylingContent?: IThemeMetadataObject;
```

Comment:
replace IThemeMetadataObject type by StylingPickerItem, that will be later extended to 
export type StylingPickerItem = IThemeMetadataObject | IColorPaletteMetadataObject;

and types will not be affected in components